Mathlib/Algebra/Module/LinearMap/Lemmas.lean

```diff
@@ -0,0 +1,41 @@
+/-
```

Comment:
Maybe the file should be called `DivisionRing.lean` or something, right now the name sounds like a file containing basic lemmas about linear maps. 